### ISSUE TITLE
Support different contexts and kubeconfig files

### ISSUE THREAD

**Issue by asimpleidea** [CloudNativeSDWAN/egress-watcher#26]
As per title, this feature will be supported in future versions as CLI have the feature but is now being ignored.

**Comment by arnatal** [comment]
Is this still being ignored? It's part of the example that appears when running the CLI without arguments.

**Comment by asimpleidea** [comment]
yes it is still ignored, will try to either have a draft for this or remove the example and re-introduce it when the feature is added
